broker-cli/commands/market/trades.js

```diff
@@ -6,11 +6,11 @@ const { handleError } = require('../../utils')
  * ex: `sparkswap market trades'
  *
  * @param {Object} args
- * @param {String} args.since timestamp for lowerbound of records, '2018-09-21T10:58:58.0131174Z'
- * @param {Integer} args.limit number of records to retrieve
+ * @param {string} args.since - timestamp for lowerbound of records, '2018-09-21T10:58:58.0131174Z'
+ * @param {Integer} args.limit - number of records to retrieve
  * @param {Object} opts
- * @param {String} [rpcaddress] opts.rpcaddress
- * @param {String} market opts.rpcaddress
+ * @param {string} [rpcaddress] - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - opts.rpcaddress
```

Comment:
this seems... odd

Reply:
hah that is super odd

Reply:
@treygriffith ive fixed these references